modules/pointcloudio/tests/test.cpp

```diff
@@ -0,0 +1,56 @@
+#include <opencv2/ts.hpp>
+#include <opencv2/core.hpp>
+#include <iostream>
+#include <vector>
+#include <string>
+
+#include "load_point_cloud.hpp"
+
+
+TEST(obj_test, point_cloud_load_tests)
+{
+    std::vector<cv::Point3f> points;
+    std::vector<cv::Point3f> normals;
+    
+    std::string folder = string(cvtest::TS::ptr()->get_data_path()) + "pointcloud/";
+    std::string original_path = folder + "orig.obj";
+    std::string new_path = folder + "new.obj";
+
+    cv::pc::loadPointCloud(original_path, points, normals);
+    
+    cv::pc::savePointCloud(new_path, points, normals);
+    
+    std::vector<cv::Point3f> points_1;
+    std::vector<cv::Point3f> normals_1;
+
+    cv::pc::loadPointCloud(new_path, points_1, normals_1);
+
+    EXPECT_EQ(1, (normals_1  == normals));
+    EXPECT_EQ(1, (points_1  == points));
```

Comment:
The test does not check the loader functionality at all. Both could be just empty. The solution works, if you have several branches in OBJ/PLY format that should be handled with the same result. The best solution here is to store ground-truth with well supported format in OpenCV like YML/JSON/XML with `cv::FileStorage` or even hard-code some small arrays in test code.